Yoyo2Code/the_pivot
# Development

* user can login

* Test user login & login features,
add dashboard to redirect successful login. Find unsuccessful redirect,
add rack session gem - very helpful.

* Add create account same page as user sign-in,
user gets redirected to dashboard after account creation. Removed mailer for now.

* Add roles table and business roles join table,
business controller - create and new routes also. Business role gets assigned via PORO after user
submits application.

* Moved user new/edit form into partial,
user can edit username - but we should consider different approach for changing password.

* item delete

* Redesign root gives user option of biz or location index,
add image_url column to business.

* user can remove multiple items from cart

* remove from cart
